README.md

```diff
@@ -12,6 +12,7 @@
 [![Javadoc](http://www.javadoc.io/badge/org.takes/takes.svg)](http://www.javadoc.io/doc/org.takes/takes)
 [![codebeat badge](https://codebeat.co/badges/5721bba4-59cd-44ea-a60f-6043734187f7)](https://codebeat.co/projects/github-com-yegor256-takes)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/938e7683fcfa4db9bff362c2b4a9b78e)](https://www.codacy.com/app/github_90/takes?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=yegor256/takes&amp;utm_campaign=Badge_Grade)
+![nullfree status](https://iwillfailyou.com/nullfree/yegor256/takes)
```

Comment:
@nikialeksey Please also add the command line parameter to the `mvn clean install -Pqulice`  in the `How to contribute` section

Reply:
Thanks! Done